ozaria/site/styles/play/level/tome/problem_alert.sass

```diff
@@ -8,16 +8,26 @@
   // Position these at the end of the spell editor, right above the spell toolbar.
   top: 45px
   right: 500px
-  background: transparent
-  border: 1px solid transparent
-  padding: 0
-  font-size: 18px
+  background-color: #ffffff
+  padding: 18.5px 13px
+  font-size: 16px
+  font-family: "Open Sans"
+  letter-spacing: 0.63px
+  line-height: 24px
   text-shadow: none
-  color: white
+  color: #232323
   word-wrap: break-word
-  border-style: solid
-  border-image: url(/images/level/code_editor_error_background.png) 16 20 fill round
-  border-width: 16px 20px
+  max-width: 342px
+
+  &::after
```

Comment:
Just curious what this selector is doing, and why it was necessary to move css into it.
Function of the transparent 10px border?

Reply:
This is to add an arrow to the right (::after) of the div. 10px transparent border adds an invisible border and then border-left-color colors it in the shape of the arrow.